store/daystore/day_store_test.go

```diff
@@ -0,0 +1,376 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	ss "k8s.io/heapster/store/statstore"
```

Comment:
No, #478 uses this daystore.

On Fri, Aug 14, 2015 at 1:54 PM, Daniel Martí <notifications@github.com>
wrote:

> In store/daystore/day_store_test.go
> <https://github.com/kubernetes/heapster/pull/471#discussion_r37118362>:
>
> > +//
> > +// Unless required by applicable law or agreed to in writing, software
> > +// distributed under the License is distributed on an "AS IS" BASIS,
> > +// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +// See the License for the specific language governing permissions and
> > +// limitations under the License.
> > +
> > +package store
> > +
> > +import (
> > +	"testing"
> > +	"time"
> > +
> > +	"github.com/stretchr/testify/assert"
> > +
> > +	ss "k8s.io/heapster/store/statstore"
>
> Wait, so should this be closed in favour of #478
> <https://github.com/kubernetes/heapster/pull/478>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/heapster/pull/471/files#r37118362>.
>